internal/bminventory/inventory.go

```diff
@@ -491,10 +491,22 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 	url := installer.V2GetClusterURL{ClusterID: id}
 
 	log := logutil.FromContext(ctx, b.log).WithField(ctxparams.ClusterId, id)
-	log.Infof("Register cluster: %s with id %s and params %+v", swag.StringValue(params.NewClusterParams.Name), id, params.NewClusterParams)
+
+	// Masked the Pull Secret to prevent its value from being displayed in the logs.
+	clusterParamsNoPullSecret := *params.NewClusterParams
+	if clusterParamsNoPullSecret.PullSecret != nil {
+		*clusterParamsNoPullSecret.PullSecret = "***"
+	}
```

Comment:
What have we done until now? It was displayed on the logs?

Reply:
@eliorerz The address was displayed